ajax/libs/bootswatch/package.json

```diff
@@ -16,10 +16,9 @@
   "npmName": "bootswatch",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "dist",
       "files": [
-        "fonts/*",
-        "*/*.css"
+        "*.css"
```

Comment:
Let's use `**/!(*.scss)`
Because the pattern `*.css` can't get the files which under subfolder.
Thanks